modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -1347,6 +1347,7 @@ def stack(self, level, dropna):
     abs = Map.register(pandas.DataFrame.abs, dtypes="copy")
     applymap = Map.register(pandas.DataFrame.applymap)
     conj = Map.register(lambda df, *args, **kwargs: pandas.DataFrame(np.conj(df)))
+    convert_dtypes = Map.register(pandas.DataFrame.convert_dtypes)
```

Comment:
Convert_dtypes requires full column access in order to determine the most appropriate dtype right? Doesn't that mean we have to do a apply_axis_partitions call on our Modin frame rather than relying on the algebra map which is cell wise?

Reply:
Sorry - just saw your discussion with Mahesh - this is just a rehash of that, so feel free to ignore/resolve